examples/dist/my-first-outreach-extension.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
```

Comment:
That's cool! It also hit my mind recently that it would be great to have an example app in the sdk repo. Next step could be to deploy it somewhere every time there is a new sdk version.